core/scripts/liquidity-mining/CalculateBalancerLPProviders.js

```diff
@@ -150,7 +150,7 @@ function _saveShareHolderPayout(shareHolderPayout, week) {
     else shareHolderPayout[shareHolder] = fromWei(shareHolderPayout[shareHolder]);
   }
 
-  const savePath = `${path.resolve(__dirname)}/weekly-payouts/${week}_week_UMAsToDistribute.json`;
+  const savePath = `${path.resolve(__dirname)}/weekly-payouts/Week_${week}_Mining_Rewards.json`;
```

Comment:
Are these the official payouts or a draft payout? If draft perhaps we should add draft to name or put in a draft dir

Reply:
this is the official payouts generated for the first week.